features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.group.create/public/js/group-add.js

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+$(function () {
+    $("button#add-group-btn").click(function () {
+
+        var name = $("input#name").val();
+        var description = $("input#description").val();
+
+        if (!name) {
+            $('.wr-validation-summary strong').text("Group Name is a required field. It cannot be empty.");
+            $('.wr-validation-summary').removeClass("hidden");
+            return false;
+        } else {
+            var addGroupAPI = "../api/group/add";
+            var group = {"name": name, "description": description};
+            var postRequest = $.ajax({
+                                         url: addGroupAPI,
+                                         method: "POST",
+                                         contentType: "application/json",
+                                         accept: "application/json",
+                                         data: JSON.stringify(group)
+                                     });
+
+            postRequest.done(function (data, textStatus, jqxhr) {
+                //var successCallback = function (data, jqxhr) {
+                var status = jqxhr.status;
+                if (status == 200) {
+                    $('.wr-validation-summary strong').text("Group created. You will be redirected to groups");
+                    $('.wr-validation-summary').removeClass("hidden");
+                    $('.wr-validation-summary strong').removeClass("label-danger");
+                    $('.wr-validation-summary strong').addClass("label-success");
+                    setTimeout(function () {
+                        window.location = "../groups";
+                    }, 1500);
+                } else {
+                    displayErrors(jqxhr);
+                }
+                //};
+            });
+
+            postRequest.fail(function (jqXHR, textStatus) {
+                displayErrors(jqXHR);
+            });
+
+            //invokerUtil.post("/common/group_manager/group", group,
```

Comment:
Acknowledged and fixed with commit https://github.com/wso2/carbon-device-mgt/commit/3dc23c5708fbbd4dab63878611b30b5e7bed44d0